### DISCUSSION TITLE
Please bring back custom misconfiguration policy docs

### DISCUSSION THREAD

**Discussion by brsolomon-deloitte** [aquasecurity/trivy#5756] [Documentation]
### Description

Previous versions of trivy documentation ([example](https://aquasecurity.github.io/trivy/v0.19.2/misconfiguration/custom/examples/)) contained examples of, or links to examples of, custom policies for multiple different types of inputs. This was way back when Trivy supported Terraform Plan files with OPA.

Current Trivy documentation [contains no such examples](https://aquasecurity.github.io/trivy/v0.48/docs/scanner/misconfiguration/custom/) and it is difficult to get started writing custom Rego for Terraform given that only a small subset of trivy/AVD cloud policies are written in Rego.

```
$ find . -name '*.rego' -type f
./aws/s3/enable_logging_test.rego
./aws/s3/dns_compliant_name_test.rego
./aws/s3/dns_compliant_name.rego
./aws/s3/enable_logging.rego
./aws/iam/filter_iam_pass_role_test.rego
./aws/iam/filter_iam_pass_role.rego
./aws/rds/enable_iam_auth_test.rego
./aws/rds/enable_cluster_deletion_protection_test.rego
./aws/rds/enable_deletion_protection_test.rego
./aws/rds/disable_public_access_test.rego
./aws/rds/enable_iam_auth.rego
./aws/rds/enable_deletion_protection.rego
./aws/rds/disable_public_access.rego
./aws/rds/enable_cluster_deletion_protection.rego
```

### Suggestions

Please bring back better example-based documentation of custom Rego policies for cloud.

**Comment by simar7** [comment]
Tracking here https://github.com/aquasecurity/trivy/issues/4255

**Reply by itaysk** [reply]
cc @AnaisUrlichs
